### DISCUSSION TITLE
Zapier integration

### DISCUSSION THREAD

**Discussion by olegcherev** [devactivity-app/devActivity-Community#22] [Ideas & Feature Requests]
Triggers:
- level achievement
- challenge completion
- alert

**Comment by andyb1979** [comment]
This would allow MS teams integration

**Reply by olegcherev** [reply]
> This would allow MS teams integration

Can you please describe a bit more about MS Teams integration? Do you need an alerts and reports sent there?

**Comment by andyb1979** [comment]
Yes it would be really useful to output alerts and notifications say to a
MS teams channel or teams chat. This can be done with Zapier

On July 12, 2025, GitHub ***@***.***> wrote:

This would allow MS teams integration

Can you please describe a bit more about MS Teams integration? Do you need
an alerts and reports sent there?

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/devactivity-app/discussions/22#discussioncomment-13738998>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ADLEDVICNXNTWYMXRN66CQ33IECKPAVCNFSM6AAAAACBLUXGB2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGNZTHA4TSOA>
.
You are receiving this because you commented.Message ID:
<devactivity-app/devActivity-Community/repo-discussions/22/comments/13738998
@github.com>
